bot.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
 date >> ~/logs/bytebot.log
-head -2 .post-queue | timeout 300 ./target/release/bot >> ~/logs/bytebot.log 2>&1
-sed -ie '1d;2d' .post-queue
+number=$RANDOM
+let "number %= 4"
+if [ "$number" -eq 0 ]
+then
+    echo "random"
+    ./target/release/random | tee ~/logs/bytebot.log 2>&1 | ./target/release/bot
```

Comment:
We should do `./target/release/random | python3 gencol.py '#bbrandom' | ./target/release/bot >> ~/logs/bytebot.log 2>&1` and fix `gencol.py` so that it will read a hashtag from the command line, this way we get random colors.

Reply:
I think we don't need a `tee` since the bot prints out what script it's formatting (right?)

Reply:
We should have `timeout` on the bot, so it can't hang itself.